projects/komponentkartan/src/lib/controls/notification/notification.component.ts

```diff
@@ -7,20 +7,40 @@ import { toggleFadedState } from '../../animation';
   animations: [toggleFadedState]
 })
 export class NotificationComponent implements OnInit {
-  @HostBinding('@toggleFadedState') animate = true;
-  @Input() @HostBinding('attr.class') borderColor: string = null;
-  @Input() type: string = null;
+  @Input() standalone = false;
+  /**
+   * @deprecated
+   */
+  @Input() borderColor: string = null;
```

Comment:
Om `border-color` blir deprecated bör vi inkludera den först i nästa major. Ser däremot att vi gått runt detta på rad 38 där den fortfarande tillåts. Ta gärna upp på mötet på fredag och se vad de andra tycker.

Reply:
Kastar deprecated ett fel i kompilering eller är det en varning? Om varning är det ju toppen att lägga in det i nuvarande version. Annars håller jag med om att behöver vänta till nästa version.

Reply:
@deprecated är bara för dokumentationssyfte [https://jsdoc.app/tags-deprecated.html](https://jsdoc.app/tags-deprecated.html) 
Det skall inte bli några kompileringsfel, varningar i konsolen eller liknande av att denna är satt. Jag vet att VS code markerar väldigt bra i ts-filer om något är deprecated (line through). Tänker att det förhoppningsvis finnas något liknande för htmln